src/Components/Dummy/Dummy.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Dummy = (props) => {
+    return <h1>Dummy component</h1>;
+};
```

Comment:
If you don't need to code anything before the return statement, you can directly return the component like this:

```
const Dummy = (props) = (
  <h1>Dummy component</h1>;
);
```